source/extensions/filters/http/aws_lambda/aws_lambda_filter.cc

```diff
@@ -333,6 +333,10 @@ void Filter::dejsonizeResponse(Http::ResponseHeaderMap& headers, const Buffer::I
     return;
   }
 
+  // Use JSON as the default content-type. If the JSON response headers has a
+  // content-type key, it will override.
```

Comment:
nit: If the response headers have a different content-type set, that will be used instead.